pkg/services/unifiedSearch/service.go

```diff
@@ -0,0 +1,176 @@
+package unifiedSearch
+
+import (
+	"context"
+	"errors"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/grafana/pkg/infra/db"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/infra/tracing"
+	"github.com/grafana/grafana/pkg/registry"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+	"github.com/grafana/grafana/pkg/services/folder"
+	"github.com/grafana/grafana/pkg/services/org"
+	"github.com/grafana/grafana/pkg/services/store"
+	"github.com/grafana/grafana/pkg/services/user"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/storage/unified/resource"
+)
+
+type StandardSearchService struct {
+	registry.BackgroundService
+	cfg            *setting.Cfg
+	sql            db.DB
+	ac             accesscontrol.Service
+	orgService     org.Service
+	userService    user.Service
+	logger         log.Logger
+	reIndexCh      chan struct{}
+	features       featuremgmt.FeatureToggles
+	resourceClient resource.ResourceClient
+}
+
+func (s *StandardSearchService) IsReady(ctx context.Context, orgId int64) IsSearchReadyResponse {
+	return IsSearchReadyResponse{IsReady: true}
+}
+
+func ProvideService(cfg *setting.Cfg, sql db.DB, entityEventStore store.EntityEventsService,
+	ac accesscontrol.Service, tracer tracing.Tracer, features featuremgmt.FeatureToggles, orgService org.Service,
+	userService user.Service, folderStore folder.Store, resourceClient resource.ResourceClient) SearchService {
+	logger := log.New("searchV3")
+	s := &StandardSearchService{
+		cfg:            cfg,
+		sql:            sql,
+		ac:             ac,
+		logger:         logger,
+		reIndexCh:      make(chan struct{}, 1),
+		orgService:     orgService,
+		userService:    userService,
+		features:       features,
+		resourceClient: resourceClient,
+	}
+	return s
+}
+
+func (s *StandardSearchService) IsDisabled() bool {
+	return !s.features.IsEnabledGlobally(featuremgmt.FlagPanelTitleSearch)
+}
+
+func (s *StandardSearchService) Run(ctx context.Context) error {
+	// TODO: implement this? ( copied from pkg/services/searchV2/service.go )
+	// orgQuery := &org.SearchOrgsQuery{}
+	// result, err := s.orgService.Search(ctx, orgQuery)
+	// if err != nil {
+	// 	return fmt.Errorf("can't get org list: %w", err)
+	// }
+	// orgIDs := make([]int64, 0, len(result))
+	// for _, org := range result {
+	// 	orgIDs = append(orgIDs, org.ID)
+	// }
+	// TODO: do we need to initialize the bleve index again ( should be initialized on startup )?
+	// return s.dashboardIndex.run(ctx, orgIDs, s.reIndexCh)
+	return nil
+}
+
+func (s *StandardSearchService) TriggerReIndex() {
```

Comment:
Do we need this?

Reply:
not sure.  they had it in searchV2 so i'm assuming there was a scenario.  but it doesn't do anything yet